Makefile

```
@@ -135,19 +135,16 @@ deploy_db_migrations:
 	@$(SQITCH) --chdir mocks_schema deploy
 
 deploy_dev_data: ## deploy the database migrations with sqitch and load the data for local development
-deploy_dev_data: deploy_db_migrations
+deploy_dev_data: deploy_db_migrations deploy_prod_data
```

we'll want to guarantee the order here:
```suggestion
deploy_dev_data: | deploy_db_migrations deploy_prod_data
```